### DISCUSSION TITLE
Anyone have a schematic for the MH-Tiny88?

### DISCUSSION THREAD

**Discussion by prosper00** [SpenceKonde/ATTinyCore#666] [General]
Some quick google-ing hasn't uncovered much... are there any schematics of the MH-ET out there? Specifically, I want to look at how the USB is connected, and details of the LDO that's onboard.

**Comment by pcfreak1201** [comment]
I don't think so. The best pictures for investigation, I found, were these:
https://www.ubuy.tn/en/product/3P2P5M9EQ-aitrip-5pcs-mh-tiny-attiny88-micro-development-board-16mhz-digispark-attiny85-upgraded-nano-v3-0-atmega328-extended-compatible-for-arduino#gallery-3
May be it helps ...

**Comment by prosper00** [comment]
ah. Well, perhaps I'll just pull out the old continuity tester, and make up some schematics the hard way

Thanks!

**Reply by pcfreak1201** [reply]
Well it was announced as "Digispark-Upgrade" - so I would start with that schematic by replacing the ATTiny85 with the ATTiny88 ...

**Comment by prosper00** [comment]
That does help, thanks. The USB front-end is exactly the same, just connected to PD2 and PD1

**Reply by pcfreak1201** [reply]
Yes, should be [V-USB](https://github.com/obdev/v-usb)

**Comment by prandeamus** [comment]
Some of the issues around V-USB on the MH-ET TINY88 board were discussed here last year: https://github.com/SpenceKonde/ATTinyCore/discussions/520 so might be of some interest. A few outcomes from that discussion were that the core now supports a special flavour of ATTINY88 targets with pin numbering that matches pin numbers on the ME-ET board (because that board has dedicated a pin to the external clock). 

I believe that the micronucleus configs are now aware of the V-USB pin mappings

#define USB_CFG_IOPORTNAME D
#define USB_CFG_DMINUS_BIT 1
#define USB_CFG_DPLUS_BIT 2

I've not looked around recently, but despite claims of the hardware being "open source" I was unable to locate any schematics for it.
